src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -427,13 +427,17 @@ public void testTooLargeAddEventsSkippedUncompressedRequest() throws Exception {
     assertEquals(0, server.getRequestCount());
 
     addEventsResponse = addEventsClient.log(events).get(5, TimeUnit.SECONDS);
+    assertEquals(addEventsResponse.getStatus(), "success");
```

Comment:
These `assertEquals` params are backwards.  expected is the first param.  SonarCloud caught this as a code smell!  Impressed.

Reply:
oops, will fix it.

And indeed a good catch on a Sonar side. Need to also check if there is a way to integrate it in a local development process (e.g. pre-commit hook or similar). Would likely be too slow though.

Reply:
a1249421124612354ee6c7047eab9d310ac05057